Source/Rhetos/RhetosRuntimeContextFactory.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System.IO;
+using Autofac;
+using Rhetos.Logging;
+using Rhetos.Security;
+using Rhetos.Utilities;
+using Rhetos.Web;
+
+namespace Rhetos
+{
+    public class RhetosRuntimeContextFactory : IRhetosRuntimeContextFactory
+    {
+        public IConfigurationBuilder ResolveConfiguration(ILogProvider logProvider, string rootFolder)
+        {
+            return new ConfigurationBuilder()
+                .AddRhetosAppEnvironment(new RhetosAppEnvironment
+                {
+                    RootFolder = rootFolder,
+                    AssetsFolder = Path.Combine(rootFolder, "Generated"),
+                    // TODO: Rhetos CLI should not use LegacyPluginsFolder. Referenced plugins are automatically copied to output bin folder by NuGet. It is used by DeployPackages.exe when downloading packages and in legacy application runtime for assembly resolver and probing paths.
+                    // TODO: Set LegacyPluginsFolder to null after reviewing impact to AspNetFormsAuth CLI utilities and similar packages.
+                    LegacyPluginsFolder = Path.Combine(rootFolder, "Plugins"),
+                    LegacyAssetsFolder = Path.Combine(rootFolder, "Resources"),
+                })
+                .AddWebConfiguration(new DirectoryInfo(rootFolder).Parent.FullName);

```

Comment:
Global.asax previously used AddConfigurationManagerConfiguration instead of AddWebConfiguration, but AddWebConfiguration is necessary if we want to reuse this configuration from utilities.